sktime/clustering/tests/metrics/test_dba.py

```diff
@@ -130,6 +132,10 @@
 )
 
 
+@pytest.mark.skipif(
+    not run_test_for_class(dba),
```

Comment:
Question: `dba` is a function right? Does `run_test_for_class` still work?

Reply:
yes. Bit of a misnormer, as the feature to use it for functions was added later - it does the same thing, get the location and run an lru_cached git diff on the path.